scripts/e2e/run-e2e.sh

```diff
@@ -47,28 +50,24 @@ function InstallAGIC() {
 }
 
 function SetupSharedBackend() {
+    [[ -z "${version}" ]] && (echo "version is not set"; exit 1)
+    [[ -z "${applicationGatewayId}" ]] && (echo "applicationGatewayId is not set"; exit 1)
+    [[ -z "${identityResourceId}" ]] && (echo "identityResourceId is not set"; exit 1)
+    [[ -z "${identityClientId}" ]] && (echo "identityClientId is not set"; exit 1)
+
     # install agic with shared enabled
     helm upgrade --install agic-${version} staging/ingress-azure \
+    -f ./helm-config-with-prohibited-rules.yaml \
     --set appgw.applicationGatewayID=${applicationGatewayId} \
-    --set armAuth.type=aadPodIdentity \
     --set armAuth.identityResourceID=${identityResourceId} \
     --set armAuth.identityClientID=${identityClientId} \
-    --set rbac.enabled=true \
-    --set appgw.shared=true \
     --timeout 120s \
     --wait \
     -n agic \
     --version ${version}
 
-    # apply customized prohibited policy for blacklisted backend
-    kubectl apply -f prohibit-blacklist-service.yaml
-
     # get all the prohibited target config
-    kubectl get AzureIngressProhibitedTargets -n agic -o yaml 
-
-    # delete default prohibit-all-targets
-    # blacklist-service shall be kept after porhibited policy applied
-    kubectl delete AzureIngressProhibitedTarget prohibit-all-targets -n agic
+    kubectl get AzureIngressProhibitedTargets -n agic -o yaml
```

Comment:
why removed  the line of "kubectl delete AzureIngressProhibitedTarget prohibit-all-targets -n agic"
the broader policy by default should be deleted

Reply:
this policy is not installed when using prohibited target in helm.